Source/WebCore/svg/SVGMarkerElement.cpp

```diff
@@ -166,6 +166,12 @@ void SVGMarkerElement::setOrientToAngle(const SVGAngle& angle)
     invalidateMarkerResource();
 }
 
+void SVGMarkerElement::setOrientToAutoStartReverse()
+{
+    Ref { m_orientType }->setBaseVal(SVGMarkerOrientAutoStartReverse);
+    invalidateMarkerResource();
+}
```

Comment:
This function is not called. I think `SVGMarkerElement.idl` should have a method to call this function similar to:
```
    undefined setOrientToAuto();
    undefined setOrientToAngle(SVGAngle angle);
```

Reply:
fixed.